### ISSUE TITLE
why its laggy i even use usb thetring tho

### ISSUE THREAD

**Issue by thediamond-hub** [teamclouday/AndroidSteering#12]
device xiaomi redmi note 9 pro android 12 i use monect and it not thatt delay??

**Comment by teamclouday** [comment]
Yes there is certain latency. Most likely this software isn't as optimized like monect. If you are seeing >1s delay for each movement, maybe try to reconnect and try again. Also make sure to set steering to linear if there's related settings in the game
